packages/shared/src/libs/helpers/change-case/libs/maps/case-type-to-function.map.ts

```diff
@@ -0,0 +1,10 @@
+import { camelCase, snakeCase } from "change-case";
+
+import { type CaseType } from "../types/types.js";
+
+const caseTypeToFunction: Record<CaseType, (input: string) => string> = {
```

Comment:
If this a map, it should be just key/value object. So either move it to helpers and name it appropriately or use object instead of function here (I would prefer the second option)

Reply:
Do you mean like this?
```
const caseTypeFunctions = {
	camelCase: (input: string) => camelCase(input),
	snakeCase: (input: string) => snakeCase(input),
} as Record<CaseType, (input: string) => string>;
```

Reply:
```typescript
const caseTypeToFunction = {
	camelCase,
	snakeCase,
}
```

Reply:
Oh, now I see :cat2: